packages/core/src/Slider/index.tsx

```diff
@@ -357,6 +367,11 @@ export const Slider: FC<SliderProps> = ({
     [max, min, value]
   )
 
+  const sliderPosition = useMemo(
+    () => fraction * sliderWidth,
+    [fraction, sliderWidth]
+  )
```

Comment:
Do you need useMemo here?

Reply:
In this instance I believe so since we will be rendering the value in a pop up every time we drag the knob. This should avoid any unnecessary re-renders.